routers/repo/issue.go

```diff
@@ -716,54 +717,84 @@ func UpdateIssueContent(ctx *context.Context) {
 	})
 }
 
-// UpdateIssueMilestone change issue's milestone
-func UpdateIssueMilestone(ctx *context.Context) {
-	issue := getActionIssue(ctx)
-	if ctx.Written() {
-		return
-	}
+// UpdateIssues perform update on collection of issues
+func UpdateIssues(ctx *context.Context) {
```

Comment:
So do you want
- separate functions for milestones/labels/assignees, each of which can handle both a single or multiple issues?
- keep the pre-existing functions milestones/labels/assignees (which can only handle a single issue), and introduce 3 new milestones/labels/assignees for multiple issues?

Also, what is wrong with what I did? I don't mean to argue, I just want to learn.